docs/guides/cloud-setup.mdx

```diff
@@ -26,6 +26,9 @@ Your user account is associated with one or more [instances](./instances) that g
 ## Set up to use IBM Cloud
 
 1. Before setting up with IBM Cloud, ensure you are working in an active Python environment with the [Qiskit SDK and Qiskit Runtime installed](/docs/guides/install-qiskit#local).
+
+1. **Users who have been invited by their organization to join an account:** Be sure you have accepted the invitation you received by email, and have created your user ID on IBM Cloud. (If you already have an IBM Cloud account associated with the invited email address, there is no need to create a new user ID.) You have already been added to an instance in your organization's account, so you can skip the "Create an instance" step and go straight to [Find your access credentials](#access-credentials).
```

Comment:
```suggestion
1. **Users who have been invited by their organization to join an account:** Be sure you have accepted the invitation you received by email and have created your user ID on IBM Cloud. If you already have an IBM Cloud account associated with the invited email address, there is no need to create a new user ID. You can skip the "Create an instance" step and go straight to [Find your access credentials](#access-credentials).
```

If you want to keep "already been added to an instance", I recommend putting it at the end as "because you have already been added to an instance". I think that may be more clear